examples/mock-app/.env.example

```
@@ -0,0 +1,13 @@
+# Test user configuration (for development only)
+TEST_USER_ID=test-user-1
+TEST_USER_EMAIL=user@example.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The TEST_USER_EMAIL key should go before the TEST_USER_ID key

(UnorderedKey)